src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -112,6 +112,7 @@ class MarathonSchedulerService @Inject() (
   def cancelDeployment(id: String): Unit =
     schedulerActor ! CancelDeployment(id)
 
+  @deprecated("Use GroupManager.root.transitiveApps", "0.10.0")
```

Comment:
why not delete this method?

It is slightly strange to use the AppRepository for versioning and nothing else, but hey!